aspnet/migration/http-modules.rst

```diff
@@ -1,8 +1,584 @@
-.. include:: /../common/stub-topic.txt
+Migrating HTTP Modules to Middleware
+====================================
 
-|stub-icon| Migrating HTTP Modules to Middleware
-======================================================
+By `Matt Perdeck`_
 
-.. include:: /../common/stub-notice.txt
```

Comment:
How about just: This article shows how to migrate existing \`HTTP modules and handlers \<https://msdn.microsoft.com/en-us/library/bb398986.aspx \>\`_ to ASP.NET 5 :ref:\`middleware \<fundamentals-middleware\>\`.